config/action.d/shorewall-ipset-proto6.conf

```diff
@@ -0,0 +1,84 @@
+# Fail2Ban configuration file
+#
+# Author: Eduardo Diaz
+#
+# This is for ipset protocol 6 (and hopefully later) (ipset v6.14).
+# for shorewall
+#
+# Use this setting in jail.conf to modify the name and the max time in every jain
+# action   = shorewall-ipset-proto6[name=SSH, bantime=10000]
```

Comment:
correct but! -- you can use string interpolations to pass those parameters borrowing variables values, thus we have
```
action_ = %(banaction)s[name=%(__name__)s, bantime="%(bantime)s", port="%(port)s", protocol="%(protocol)s", chain="%(chain)s"]
```
so it should be enough to say "banaction = shorewall-ipset-proto6".  I will do that change